src/main/webapp/style.css

```diff
@@ -4,4 +4,19 @@
   position: absolute;
   top: 0px;
   left: 0px;
-}
\ No newline at end of file
+}
+
+#search {
+  position: absolute;
+  top: 10px;
```

Comment:
This isn't really a big deal, but I noticed when doing the portfolio project that using px makes the page look weird on different sized screens. The way I fixed it was by using a % instead of px.

Reply:
The problem I had with % in this case was that the Map and Satellite buttons dont change size so using % on the search bar could make it overlap sometimes.